skyplane/replicate/replicator_client.py

```diff
@@ -609,11 +609,16 @@ def monitor_transfer(
                         # update progress bar
                         total_runtime_s = (log_df.time.max() - log_df.time.min()).total_seconds()
                         throughput_gbits = completed_bytes * 8 / GB / total_runtime_s if total_runtime_s > 0 else 0.0
+                        pending_chunk_ids = set([cr.chunk.chunk_id for cr in job.chunk_requests]) - set(completed_chunk_ids)
+                        if len(pending_chunk_ids) < 5:
+                            pending_chunks = ", chunks [" + ", ".join([str(cid) for cid in pending_chunk_ids]) + "] remain"
```

Comment:
May not be necessary, re comment

Reply:
I'll only enable this if `debug` is passed.